### PR TITLE
NER example for Tensorflow

### PR DESCRIPTION
**Pull request by Rocketknight1**

### CODE REVIEW

File: examples/tensorflow/token_classification/run_ner.py [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+            tf_train_dataset,
+            validation_data=tf_eval_dataset,
+            epochs=int(training_args.num_train_epochs),
+            steps_per_epoch=train_batches_per_epoch,
+            validation_steps=eval_batches_per_epoch,
+        )
+
+    # We don't do predictions in the strategy scope because there are some issues in there right now.
+    # They'll get fixed eventually, promise!
+
```
Comment (sgugger) [inline examples/tensorflow/token_classification/run_ner.py:471]: Can we add a quick and dirty evaluation that gets the result of predict and passes it to `metric.compute()`? A bit like the `compute_metrics` function in the PyTorch example.

**Review by sgugger — APPROVED** [review]
Thanks for adding this example!

File: examples/tensorflow/token_classification/run_ner.py (hunk 1 above)
Comment (Rocketknight1) [reply examples/tensorflow/token_classification/run_ner.py:471]: I can probably do that quickly tomorrow, sure!

**Review by LysandreJik — APPROVED** [review]
Great, thanks for working on it!
